README.md

```diff
@@ -8,6 +8,179 @@ See https://github.com/projectnessie/nessie
+  id("org.projectnessie") version "0.29.0"
+}
+
+dependencies {
+  // your dependencies
+
+  // specify the GAV of the Nessie Quarkus server runnable (uber-jar)
+  nessieQuarkusServer("org.projectnessie:nessie-quarkus:0.49.0:runner")
+}
+
+nessieQuarkusApp {
+  // Ensure that the `test` task has a Nessie Server available.  
+  includeTask(tasks.named<Test>("test"))
+  // Note: prefer setting up separate `integrationTest` tasks, see https://docs.gradle.org/current/userguide/jvm_test_suite_plugin.html
+
+  // These system properties will be available in your integration tests and will contain the
+  // port and full URL of the Nessie Quarkus server's HTTP server.
+  // httpListenPortProperty.set("quarkus.http.port") // quarkus.http.port is the default
+  // httpListenUrlProperty.set("quarkus.http.test-url") // quarkus.http.test-url is the default
+}
+```
+
+#### Groovy DSL
+
+`build.gradle`
+```groovy
+plugins {
+  id 'java'
+  id 'org.projectnessie' version "0.29.0"
+}
+
+dependencies {
+  // your dependencies
+
+  // specify the GAV of the Nessie Quarkus server runnable (uber-jar)
+  nessieQuarkusServer "org.projectnessie:nessie-quarkus:0.49.0:runner"
+}
+
+nessieQuarkusApp {
+  // Ensure that the `test` task has a Nessie Server available.  
+  includeTask(tasks.named("test"))
+  // Note: prefer setting up separate `integrationTest` tasks, see https://docs.gradle.org/current/userguide/jvm_test_suite_plugin.html
+
+  // These system properties will be available in your integration tests and will contain the
+  // port and full URL of the Nessie Quarkus server's HTTP server.
+  // httpListenPortProperty.set("quarkus.http.port") // quarkus.http.port is the default
+  // httpListenUrlProperty.set("quarkus.http.test-url") // quarkus.http.test-url is the default
+}
+```
+
+### Maven
+
+The `org.projectnessie:nessie-apprunner-maven-plugin` Maven plugin can be used together with the
+standard `maven-failsafe-plugin`
+
+(POM snippet)
+
+```xml
+<project>
+
+  <properties>
+    <nessie.version>0.49.0</nessie.version>
+  </properties>
+  
+  <dependencies>
+    <dependency>
+      <groupId>org.projectnessie</groupId>
+      <artifactId>nessie-client</artifactId>
+      <version>${nessie.version}</version>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-failsafe-plugin</artifactId>
+      </plugin>
+
+      <plugin>
+        <groupId>org.projectnessie</groupId>
+        <artifactId>nessie-apprunner-maven-plugin</artifactId>
+        <version>0.29.0</version>
+        <configuration>
+          <!-- Preferred way, specify the GAV of the Nessie Quarkus server runnable (uber-jar) -->
+          <appArtifactId>org.projectnessie:nessie-quarkus:jar:runner:${nessie.version}</appArtifactId>
+          <!-- The system properties passed to the Nessie server -->
+          <systemProperties>
+            <foo>bar</foo>
+          </systemProperties>
```

Comment:
systemPropertyVariables ? (same change as you did to the other poms)

Reply:
That's a property defined by our plugin